src/components/profiles/profiles.js

```diff
@@ -0,0 +1,44 @@
+import { getCloudinaryUrl } from '../../plugins/cloudinary/common';
+import { fetchWithTimeout } from '../../utils/fetch-with-timeout';
+
+const defaultProfilesNames = ['loopingVideo', 'videoStream', 'stylish'];
+
+export const getDefaultProfile = async (profileName) => {
+  const defaultProfiles = {
+    loopingVideo: await import(/* webpackChunkName: "profiles/loopingVideo" */ '../../profiles/loopingVideo.json'),
+    videoStream: await import(/* webpackChunkName: "profiles/videoStream" */ '../../profiles/videoStream.json'),
+    stylish: await import(/* webpackChunkName: "profiles/stylish" */ '../../profiles/stylish.json')
+  };
```

Comment:
If you load this whole module lazily you wouldn't need to do that

Reply:
good suggestion, changed it 👍